src/test/scala/uk/gov/homeoffice/domain/core/email/EmailRepositorySpec.scala

```diff
@@ -3,11 +3,20 @@ package uk.gov.homeoffice.domain.core.email
 import org.bson.types.ObjectId
 import org.joda.time.DateTime
 import org.specs2.mutable.Specification
-import uk.gov.homeoffice.mongo.casbah.MongoSpecification
+import org.specs2.specification.AfterEach
 import uk.gov.homeoffice.domain.core.email.EmailStatus._
+import uk.gov.homeoffice.mongo.TestMongo
+
+class EmailRepositorySpec extends Specification with AfterEach {
+  val repository = EmailRepository(TestMongo.testConnection)
+
+  sequential
+
+  def after() :Unit = {
+    println("dropping db contents between unit tests")
```

Comment:
can remove println here

Reply:
Removed `AfterEach` and put the repository in `Context` so that it changes between tests instead which I think is cleaner.